crates/sui-graphql-rpc/src/types/move_package.rs

```diff
@@ -212,8 +218,10 @@ impl MovePackage {
         db: &Db,
         address: SuiAddress,
         version: Option<u64>,
+        checkpoint_sequence_number: Option<u64>,
```

Comment:
nit: given a package is an immutable object, this is potentially not as useful -- we could always pass `null` to the `MoveObject::query` call.